spec/main-spec.js

```diff
@@ -71,6 +79,13 @@ describe('SSH2', function() {
       expect('Should not have hit').toBe(error)
     }
   })
+  sshit('connects to a server with a inline private key', async function(port, client) {
+    try {
```

Comment:
If we want it to succeed, then you can just remove the try/catch block.
By default the error will fail if awaiting on it fails